tests/installation/await_install.pm

```diff
@@ -123,6 +123,8 @@ sub run {
     if (check_var('ARCH', 's390x')) {
         push(@tags, 'additional-packages');
     }
+    # For poo#64228, we need ensure the timeout value less than the MAX_JOB_TIME
+    $timeout = get_var('MAX_JOB_TIME') - 1000 if ($timeout > get_var('MAX_JOB_TIME'));
```

Comment:
I would prefer a hard failure early in the tests if MAX_JOB_TIME is not high enough instead of silently lowering the timeout. This can be very hard to follow for people why above timeout is set to big values and then we silently set it lower values here. How about:

```perl
my $max_job_time_bound = get_required_var('MAX_JOB_TIME') - 1000;
die "Computed timeout '$timeout' exceeds max_job_time_bound '$max_job_time_bound', consider decreasing '$timeout' or increasing 'MAX_JOB_TIME'" if $timeout > $max_job_time_bound;
```

This way it will be very explicit what needs to be changed.

Reply:
Understand it, I updated code following your suggestion. I will run a verify log for it.

Reply:
looks good, thx. Waiting for the verification run.